server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultConnectOptionsContext.java

```diff
@@ -222,6 +242,10 @@ public DefaultConnectOptionsContext(ServiceConnectOptionsType connectOptions, Se
         }
 
         this.sslEncryptionEnabled = (sslEncryptionEnabled == null) ? true : sslEncryptionEnabled;
```

Comment:
Perhaps you could set the default value on the field at the top of the file and then override the value as it is set, thereby no longer needing the tmp variables.  I get whats going on here, but its a slightly different pattern then what other properties are doing in this class, see `sslEncryptionEnabled = true; // default to true`.  Also @krismcqueen is reworking this class in general so you might want to sync with him